reactioncommerce/reaction
# refactor: use npm package for files plugin

Resolves https://github.com/reactioncommerce/reaction/issues/6172

This PR removes the internal `files` plugin, and installs the new `api-plugin-files` from npm as a replacement.

The code in that npm plugin is the same, so the expected outcome of this PR is seeing nothing different than how the app currently runs.

## Testing
- Start reaction, see that `files` is the correct version in the `systemInformation` query
- Upload a file through Admin and verify it's working

Signed-off-by: Maria Paktiti <maria.paktiti@gmail.com>
